src/App.js

```diff
@@ -19,9 +19,9 @@ function App() {
       <About />
       <Venue /> 
       <Partners />
-      {/* <Speakers />
+      {/* <Speakers />*/
```

Comment:
I guess moving this below `<Venue />` and changing the background colour to match

Reply:
Is this intended? There is a image in the second row...

![Screenshot_2019-08-01 DevConZM19](https://user-images.githubusercontent.com/12296763/62280538-6c5c1900-b44c-11e9-8d29-21d881630fd1.png)

Reply:
Saw it in the master build actually. I figured it was a broken link to one of the partners logos or something. Didn't check the code for the partners section

Reply:
I have just seen it too... I'll fix it...

Reply:
So @malgamves what have we decided on where the speakers and the venue goes. @k9uma I think that's what's holding this up

Reply:
> > Thanks, a few big design changes coming from #107 so will be best to merge this after that.
> > I think it's worth adding ticketing above partners for visibility. For contrast, it would pay to add the banners just below the "9 to 5 o'clock" so they take up the grey background.
> 
> I agree with you on this, if we push it up a bit it maybe above partners it would be more visible.

We will go with what @OlivierJM said with the Venue and them the speaker page will be in a separate page. We will need react router for that

Reply:
@malgamves if this is done, let's mark it as resolved, I think we will need to create another issue for the Code of Conduct and speakers, as you mentioned we will need a router for that.